docs/gadget-devel/gadget-wasm-api-raw.md

```diff
@@ -20,7 +20,7 @@ contains the length and the lower 32 the memory address.
 
 ## Wasm Module Exported Functions
 
-The Wasm module implemented by the gadget also needs to export some functions to
+The `ig` wasm module implemented by the gadget also needs to export some functions to
 be invoked by the host.
```

Comment:
Functions like `newDataSource` are **imported** in the gadget wasm module, so the "ig" name is implemented in the wasm host and not by the gadget.

Could this be rephrased accordingly?

Reply:
Seems like I totally changed the wrong place in the documentation. Fixed it